.github/workflows/mpet-docker-test.yml

```diff
@@ -0,0 +1,65 @@
+name: MPET docker CI
+
+on:
+  pull_request:
+    branches: [ master,development ]
+
+jobs:
+  test:
+    
+    runs-on: ubuntu-latest
+    container:
+      image: vikko00/mpet-deps:latest
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
What's the reason for the multiple checkouts?

Reply:
There are three checkouts:
- the checkout@v1 also includes the .git directory, which is necessary for coveralls to link it. 
- the two checkout@v2 only do a "shallow" checkout and they do not include the .git repository, technically they can be created from the checkout@v2, but for clarity they are separated.